gradle.properties

```diff
@@ -0,0 +1,33 @@
+org.gradle.caching=true
+puppycrawlCheckstyleVersion=8.18
+group=org.ballerinalang
+version=0.1.0-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha6-SNAPSHOT
```

Comment:
Shall we update to the new language and stdlib versions

Reply:
Resolved with 171baaf97be39b48c6663ec8edeb3d05944f98f0 and 96e6a5cf8f919db53bdd3480ba1629df082e61fc